server/historian/packages/historian-base/src/logger.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as debug from "debug";
+import * as winston from "winston";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+import Transport = require("winston-transport");
+
+export interface IWinstonConfig {
+    colorize: boolean;
+    json: boolean;
+    label: string;
+    level: string;
+    timestamp: boolean;
+    additionalTransportList: Transport[];
+}
```

Comment:
Not for this PR, but we should copy this inside a shared library and have all services consume it.